superset-frontend/js_build.sh

```diff
@@ -20,6 +20,7 @@ cd "$(dirname "$0")"
 npm --version
 node --version
 time npm ci
-time npm run lint
+time npm run eslint -- .
```

Comment:
`--` from my understanding is a `npm run` feature where it allows you to pass arguments to the underlying command. npm will basically concatenate what's on the right of `--` with the script / string you define. 

In this case, I designed the `npm run eslint` to not include the `.` like it did before, so that a user that knows these conventions could run `npm run eslint -- myfile.js` or `npm run eslint -- --fix .`. Somehow `npm run type` script is `tsc --noEmit`, it doesn't require a file argument like eslint